### PR TITLE
[13.0][MIG] knowledge: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT20475

### CODE REVIEW

File: knowledge/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Knowledge",
+    "version": "13.0.1.0.0",
+    "author": "OpenERP SA,"
```
Comment (Tardo) [inline knowledge/__manifest__.py:6]: Seems outdated...

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
One little comment: please add in data the permission "Knowledge user" to "Mitchell Admin" user

File: knowledge/models/res_config.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class KnowledgeConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    module_document = fields.Boolean(
```
Comment (Tardo) [inline knowledge/models/res_config.py]: In 13.0 the module is called: attachment_indexation

**Review by Tardo — CHANGES_REQUESTED** [review]

File: knowledge/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply knowledge/__manifest__.py:6]: That was the author in the original module

File: knowledge/models/res_config.py (hunk 2 above)
Comment (pedrobaeza) [reply knowledge/models/res_config.py]: Good catch!

Comment (ernestotejeda) [reply knowledge/models/res_config.py]: :+1: Done

**Comment by ernestotejeda** [comment]
Changes done

**Review by Tardo — APPROVED** [review]
LGTM :+1:

**Pull request merged**